src/common/schema/application.js

```diff
@@ -0,0 +1,293 @@
+      type: 'boolean'
+    },
+    isAsian: {
+      type: 'boolean'
+    },
+    isWhite: {
+      type: 'boolean'
+    },
+    veteranAddress: {
+      $ref: '#/definitions/address'
+    },
+    email: {
+      type: 'string',
+      format: 'email'
+    },
+    homePhone: {
+      $ref: '#/definitions/phone'
+    },
+    mobilePhone: {
+      $ref: '#/definitions/phone'
+    },
+    provideFinancialInfo: {
+      type: 'boolean'
+    },
+    understandsFinancialDisclosure: {
+      type: 'boolean'
+    },
+    spouseFullName: {
+      $ref: '#/definitions/fullName'
+    },
+    spouseSocialSecurityNumber: {
+      $ref: '#/definitions/ssn'
+    },
+    spouseDateOfBirth: {
+      $ref: '#/definitions/date'
+    },
+    dateOfMarriage: {
+      $ref: '#/definitions/date'
+    },
+    sameAddress: {
+      type: 'boolean'
+    },
+    cohabitedLastYear: {
+      type: 'boolean'
+    },
+    provideSupportLastYear: {
+      type: 'boolean'
+    },
+    spouseAddress: {
+      $ref: '#/definitions/address'
+    },
+    spousePhone: {
+      $ref: '#/definitions/phone'
+    },
+    children: {
+      type: 'array'
+    },
+    veteranGrossIncome: {
+      $ref: '#/definitions/monetaryValue'
+    },
+    veteranNetIncome: {
+      $ref: '#/definitions/monetaryValue'
+    },
+    veteranOtherIncome: {
+      $ref: '#/definitions/monetaryValue'
+    },
+    spouseGrossIncome: {
+      $ref: '#/definitions/monetaryValue'
+    },
+    spouseNetIncome: {
+      $ref: '#/definitions/monetaryValue'
+    },
+    spouseOtherIncome: {
+      $ref: '#/definitions/monetaryValue'
+    },
+    deductibleMedicalExpenses: {
+      $ref: '#/definitions/monetaryValue'
+    },
+    deductibleFuneralExpenses: {
+      $ref: '#/definitions/monetaryValue'
+    },
+    deductibleEducationExpenses: {
+      $ref: '#/definitions/monetaryValue'
+    },
+    isCoveredByHealthInsurance: {
+      type: 'boolean'
+    },
+    isMedicaidEligible: {
+      type: 'boolean'
+    },
+    isEnrolledMedicarePartA: {
+      type: 'boolean'
+    },
+    medicarePartAEffectiveDate: {
+      $ref: '#/definitions/date'
+    },
+    lastServiceBranch: {
+      'enum': options.branchesServed.map(option => option.value)
+    },
+    lastEntryDate: {
+      $ref: '#/definitions/date'
+    },
+    lastDischargeDate: {
+      $ref: '#/definitions/date'
+    },
+    dischargeType: {
+      'enum': options.dischargeTypes.map(option => option.value)
+    },
+    purpleHeartRecipient: {
+      type: 'boolean'
+    },
+    isFormerPow: {
+      type: 'boolean'
+    },
+    postNov111998Combat: {
+      type: 'boolean'
+    },
+    disabledInLineOfDuty: {
+      type: 'boolean'
+    },
+    swAsiaCombat: {
+      type: 'boolean'
+    },
+    vietnamService: {
+      type: 'boolean'
+    },
+    exposedToRadiation: {
+      type: 'boolean'
+    },
+    radiumTreatments: {
+      type: 'boolean'
+    },
+    campLejeune: {
+      type: 'boolean'
+    }
+  }
+};
```

Comment:
Without verifying that this is 100% schematically correct, I will say that this is awesome a big improvement 👍